tests/ft_integration/Tribe/Tickets/__snapshots__/MetaboxTest__test_get_panels__post with tickets and RSVPs__0.snapshot.html

```diff
@@ -648,8 +648,10 @@ <h4>Ticket Settings	</h4>
 			class="ticket_field ticket_form_right"
 			size="7"
 			value=""
-						data-validation-error="Ticket price must be greater than zero." data-required data-validation-is-greater-than="0"		/>
-		<div class="ticket_sale_price_wrapper ticket_form_right">
+						data-validation-error="Ticket price must be greater than zero."		/>
+					<p class="description ticket_form_right">
```

Comment:
These look inconsistent,  if the `data-required data-validation-is-greater-than="0"` validation is not happening then the `data-validation-error` should also be removed.